3cqsbot.py

```diff
@@ -281,9 +282,10 @@ async def my_event_handler(event):
         logging.info("Bot stopped - no new signals processed")
     else:
 
-        logging.debug("New signals incoming...")
+        logging.info("New 3CQS signals incoming...")
 
         tg_output = tg_data(parse_tg(event.raw_text))
+        logging.info("TG msg: "+ str(tg_output))
```

Comment:
Do we really want this to be info? This will mess up the info logging. Please make it debug.